tools/cli/pkg/ers/command/migrate_all.go

```diff
@@ -0,0 +1,153 @@
+package command
+
+import (
+	"bufio"
+	"encoding/json"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/kyma-project/control-plane/tools/cli/pkg/ers"
+	"github.com/kyma-project/control-plane/tools/cli/pkg/ers/client"
+	"github.com/pkg/errors"
```

Comment:
replace "github.com/pkg/errors" with
"errors" as this is now part auf standard library and pkg/errors was archived